modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -0,0 +1,55 @@
+package scala.cli.commands
+
+import caseapp._
+import coursier.env.{EnvironmentUpdate, ProfileUpdater}
+import scala.io.StdIn.readLine
+
+import scala.util.Properties
+
+object InstallHome extends ScalaCommand[InstallHomeOptions] {
+  override def hidden: Boolean = true
+  def run(options: InstallHomeOptions, args: RemainingArgs): Unit = {
+
+    val binDirPath      = scala.build.Directories.default().binRepoDir
+    val scalaCliBinPath = binDirPath / "scala-cli"
+
+    if (os.exists(scalaCliBinPath)) {
+      println("scala-cli already exists. Do you want to override it in local repo directory [Y/n]")
+      val replace = readLine()
```

Comment:
We don't want to call `readLine` in non-interactive environments (like CIs). This can be detected in most cases with `System.console() == null`, but it's not enough sometimes IIRC. So `coursier.paths.Util.useAnsiOutput()` could be used (it's misnamed, it's not really about ansi stuff…).

You may also want to add a `-f` / `--force` option, to force erasing things no matter what.